### PR TITLE
Update and repair the Vagrantfile

### PR DESCRIPTION
**Pull request by RaitoBezarius**
With fixed instructions.

It only requires a working seed data to completely initialize the instance.

- Add a Ruby script to insert the current user's SSH key inside the virtual machine so that `ssh root@app.mangaki.dev` works out of the box.
- Add a _virtual_ environment in the _virtual_ machine (because the system `pip` is too outdated).
- If `vagrant-hostsupdater` plugin is available, the `/etc/hosts` is updated with the IP address of the VM and `app.mangaki.dev` as a hostname.
- Allowed hosts array is `['*']` because of [recent changes introduced in Django 1.10.3](https://docs.djangoproject.com/en/1.10/ref/settings/#allowed-hosts) regarding `ALLOWED_HOSTS` and `DEBUG` interaction.

### CODE REVIEW

**Comment by jilljenn** [comment]
Échec sur :

    PermissionError: [Errno 13] Permission denied: '/usr/local/lib/python3.4/dist-packages/virtualenv.py'

Ensuite j'ai eu un truc marrant 😄 
capture d ecran 2017-02-07 a 23 52 33

**Comment by RaitoBezarius** [comment]
@jilljenn Super, merci, j'investigue ça aussitôt.

File: provisioning/bootstrap.sh [hunk 1]
```diff
@@ -36,13 +43,19 @@ DB_PASSWORD = ${DB_PASSWORD}
 EMAIL_BACKEND = django.core.mail.backends.console.EmailBackend
 EOF
 
-sudo -H -u vagrant pip3 install --user -r /vagrant/requirements.txt scikit-learn
+sudo -H -u vagrant pip3 install virtualenv
+sudo -H -u vagrant /home/vagrant/.local/bin/virtualenv --python=python3 $VIRTUALENV_PATH
```
Comment (RaitoBezarius) [inline provisioning/bootstrap.sh]: I see now why it's failing, in my head, the line before was `pip3 install --user virtualenv` which would install into a `.local` folder.
But `pip3 install virtualenv` is most likely going to fail due to PermissionErrors.

`pipenv` is a great idea!

**Comment by jilljenn** [comment]
By the way, the installation took 2.49 GB on my disk. We should mention this in the README.

**Comment by RaitoBezarius** [comment]
@jilljenn Definitely, it's most likely due to the Debian image being downloaded + installed on a virtual hard drive.

**Comment by RaitoBezarius** [comment]
@jilljenn Done, can you review/test again?

**Comment by jilljenn** [comment]
-bash: /home/vagrant/.virtualenv/bin/activate: No such file or directory

Should I start again with a fresh copy?

**Comment by RaitoBezarius** [comment]
@jilljenn Yes, please. `vagrant halt && vagrant destroy && vagrant up`.

**Comment by jilljenn** [comment]
Finally!

**Pull request merged**